tests/api_tests/test_ancillary_studies_cloud_tasks.py

```diff
@@ -105,3 +119,117 @@ def tearDown(self):
         self.clear_table_after_test("nph.enrollment_event_type")
         self.clear_table_after_test("nph.participant_event_activity")
         self.clear_table_after_test("nph.enrollment_event")
+
+
+class InsertNphIncidentTaskApiCloudTaskTest(BaseTestCase):
+
+    DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+    TIME = datetime.strptime(datetime.now().strftime(DATETIME_FORMAT), DATETIME_FORMAT)
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.nph_participant_dao = NphParticipantDao()
+        self.nph_activity_dao = NphActivityDao()
+        self.nph_participant_event_activity_dao = NphParticipantEventActivityDao()
+        self.nph_incident_dao = NphIncidentDao()
+        self.faker = Faker()
```

Comment:
Might be worthwhile to check out the Parent class sometime because `BaseTestCase` already includes certain classes that are utilized in testing such as `Faker`.